azext_edge/tests/edge/init/test_work_unit.py

```diff
@@ -339,7 +331,10 @@ def test_init_to_template_params(
     for set_value_tuple in set_value_tuples:
         assert parameters[set_value_tuple[0]]["value"] == set_value_tuple[1]
 
-    assert template_ver.content["variables"]["AIO_CLUSTER_RELEASE_NAMESPACE"] == cluster_namespace or DEFAULT_NAMESPACE
+    if cluster_namespace:
+        assert template_ver.content["variables"]["AIO_CLUSTER_RELEASE_NAMESPACE"] == cluster_namespace
+    else:
+        assert template_ver.content["variables"]["AIO_CLUSTER_RELEASE_NAMESPACE"] == DEFAULT_NAMESPACE
```

Comment:
nit: could have left it as the or with parentheses: (cluster_namespace or DEFAULT_NAMESPACE)

Reply:
I changed it to this to be more clear on expected assertions, the or could hide an issue.